dev/tasks/cherry-pick

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# This script cherry-picks (merged) PRs from the main branch to a release branch,
+# and then sends a github PR for the changes.
+#
+# It assumes that the gh CLI is installed and authenticated.
+#
+# Usage: dev/tasks/cherry-pick <release-branch> <pr-number>
+
+import os
+import subprocess
+import sys
+import tempfile
+import json
+
+def run(cmd):
+    print(f"Running command: {' '.join(cmd)}")
+    return subprocess.run(cmd, check=True, capture_output=True, text=True)
+
+
+def main():
+    if len(sys.argv) != 3:
+        print("Usage: dev/tasks/cherry-pick <release-branch> <pr-number>")
+        sys.exit(1)
+    release_branch = sys.argv[1]
+    pr_number = sys.argv[2]
+
+    repo_root = run(["git", "rev-parse", "--show-toplevel"]).stdout.strip()
```

Comment:
Seems like we should be checking the result of run().returncode is non zero and exiting.